app/js/handlebarsScoreTable.ts

```diff
@@ -10,12 +10,6 @@ import {UserAnswers} from "./interfaces/UserAnswers";
  */
 async function sortUsersObjectByDate() {
```

Comment:
this function seems redundant now. can it be removed and just call the `createUsersObject` instead?